concrete/src/Html/Image.php

```diff
@@ -45,42 +43,68 @@ public function __construct(File $f = null, $usePictureTag = null)
 
         if ($this->usePictureTag) {
             if (!isset($this->theme)) {
-                $c = \Page::getCurrentPage();
+                $c = Page::getCurrentPage();
                 $this->theme = $c->getCollectionThemeObject();
             }
-            $sources = array();
+            $sources = [];
             $fallbackSrc = $f->getRelativePath();
             if (!$fallbackSrc) {
                 $fallbackSrc = $f->getURL();
             }
             foreach ($this->theme->getThemeResponsiveImageMap() as $thumbnail => $width) {
-                $type = \Concrete\Core\File\Image\Thumbnail\Type\Type::getByHandle($thumbnail);
+                $type = Type::getByHandle($thumbnail);
                 if ($type != null) {
                     $src = $f->getThumbnailURL($type->getBaseVersion());
-                    $sources[] = array('src' => $src, 'width' => $width);
+                    $sources[] = ['src' => $src, 'width' => $width];
                     if ($width == 0) {
                         $fallbackSrc = $src;
                     }
                 }
             }
-            $this->tag = \Concrete\Core\Html\Object\Picture::create($sources, $fallbackSrc);
+            $this->tag = Picture::create($sources, $fallbackSrc);
         } else {
             // Return a simple image tag.
             $path = $f->getRelativePath();
             if (!$path) {
                 $path = $f->getURL();
             }
-            $this->tag = \HtmlObject\Image::create($path);
+            $this->tag = HtmlObjectImage::create($path);
             $this->tag->width((string) $f->getAttribute('width'));
             $this->tag->height((string) $f->getAttribute('height'));
         }
     }
 
     /**
-     * @return \HTMLObject\Element\Tag
+     * Returns an object that represents the HTML tag.
+     *
+     * @see https://github.com/Anahkiasen/html-object
+     *
+     * @return \Concrete\Core\Html\Object\Picture|\HtmlObject\Image
      */
     public function getTag()
     {
         return $this->tag;
     }
+
+    /**
+     * Load picture settings from the theme.
+     *
+     * If the theme uses responsive image maps,
+     * the getTag method will return a Picture object.
+     */
+    protected function loadPictureSettingsFromTheme()
+    {
+        $c = Page::getCurrentPage();
+        if (is_object($c)) {
```

Comment:
I'd use `if ($c && !$c->isError()) {`

Reply:
I've tried not to change any of the logic, considering it takes long before PRs are merged...

Reply:
> I've tried not to change any of the logic, considering it takes long before PRs are merged...

One more reason to include such minor fixes in just one PR (at least that's how I behave) :wink:

Reply:
I'm not following, I haven't changed the logic, right? I've only changed the order of the methods.

Reply:
> I'm not following

You wrote that you don't implement minor fixes in this PR since it takes long to merge PRs.
I answered that, exactly because it takes long to merge PRs, I do implement minor fixes in my PRs.

Reply:
I prefer to only include obvious code changes in this PR.